DatumBoxTests.py

```diff
@@ -37,7 +37,7 @@
 
 basic_english_text = "Programming is telling a computer how to do certain things by giving it instructions. These instructions are called programs. A person who writes instructions is a computer programmer. These instructions come in different languages; they are called programming languages. Sometimes programmers use special software which helps them to make programs, and sometimes they use simpler software, called a text editor, which only gives them a place to type."
 
-commercial_text = """Nexus 4 is the latest smartphone from Google. With cutting edge hardware, the latest version of Android, and the best Google apps  Nexus 4 puts the world's information at your fingertips."""
+commercial_text = """We offer commercial and corporate photography to businesses looking for experienced photographers with the skills and professionalism to meet their needs"""
```

Comment:
I don't think this PR should change the Tests. Please revert this change.

Reply:
For the above commercial_text 
```
FAIL: test_commercial (__main__.TestCommercialDetection)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "DatumBoxTests.py", line 174, in test_commercial
    self.assertEqual(datum_box.is_commercial(commercial_text), True)
AssertionError: False != True
```
That's why I changed the commercial Text string